core/src/main/java/org/sonarsource/sonarlint/core/container/global/ExtensionInstaller.java

```diff
@@ -34,24 +34,29 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(ExtensionInstaller.class);
 
-  private final SonarQubeVersion sonarQubeVersion;
+  private final SonarRuntime sqRuntime;
   private final DefaultPluginRepository pluginRepository;
 
-  public ExtensionInstaller(SonarQubeVersion sonarQubeVersion, DefaultPluginRepository pluginRepository) {
-    this.sonarQubeVersion = sonarQubeVersion;
+  public ExtensionInstaller(SonarRuntime sqRuntime, DefaultPluginRepository pluginRepository) {
+    this.sqRuntime = sqRuntime;
     this.pluginRepository = pluginRepository;
   }
 
-  public ExtensionInstaller install(ComponentContainer container, ExtensionMatcher matcher) {
+  public ExtensionInstaller install(ComponentContainer container) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "install" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)